### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by azadehag**
added a line about creator (saavy girl)

### CODE REVIEW

**Review by SAlirezaKh — APPROVED** [review]
Great work!

**Comment by SAlirezaKh** [comment]
I have a question

**Comment by azadehag** [comment]
> I have a question

yes, sure, please leave your comment on readme file

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 An introduction to github
 This is my first repository in github.
 This repository has been made for educating purpose.
+
+The reposity was created by a data-saavy student which is in the first steps of her way.
```
Comment (SAlirezaKh) [inline README.md:6]: This line is qrong

**Review by azadehag — COMMENTED** [review]
great work

**Pull request closed without merging**